core/client/ui/radial-menu.js

```diff
@@ -131,7 +131,13 @@ const buildMenuFromOptions = options => {
 const onMouseMove = event => {
   if (!Session.get('menu') || menuOpenUsingKey) return;
   const menuPosition = computeMenuPosition();
-  const mousePosition = { x: event.clientX, y: event.clientY };
+
+  // Get canvas bound go manage mouse position offset
+  const canvasBounds = document.querySelector('#game canvas').getBoundingClientRect();
```

Comment:
Is it not possible to cache this? A call to the DOM at each mouse movement is very expensive